src/services/company-officers/companyOfficers.service.ts

```diff
@@ -24,8 +25,9 @@ export default class CompanyOfficersService {
     }
 
     const activeDirectors: DirectorDetails[] = response.resource!.items
-      .filter((officer: CompanyOfficer) => officer.officerRole === 'director')
-      .filter((director: CompanyOfficer) => !director.resignedOn)
+      .filter((officer: CompanyOfficer) =>
+        (officer.officerRole === OfficerRole.DIRECTOR || officer.officerRole === OfficerRole.LLP_MEMBER) && !officer.resignedOn
```

Comment:
The valid officer roles should be moved out to a class-level constant:
```
private readonly VALID_OFFICER_ROLES: OfficerRole[] = [
  OfficerRole.DIRECTOR,
  OfficerRole.LLP_MEMBER
]
```

Also, this can be simplified:
```
.filter((officer: CompanyOfficer) => VALID_OFFICER_ROLES.includes(officer.officerRole) && !officer.resignedOn)
```